packages/editor-ui/src/store.ts

```diff
@@ -774,16 +779,32 @@ export const store = new Vuex.Store({
 		allNodeTypes: (state): INodeTypeDescription[] => {
 			return state.nodeTypes;
 		},
-		nodeType: (state) => (nodeType: string): INodeTypeDescription | null => {
+		allNodeTypesDefaultVersion: (state): INodeTypeDescription[] => {
+			return state.nodeTypesDefaultVersion;
+		},
+		nodeType: (state, getters) => (nodeType: string, typeVersion?: number): INodeTypeDescription | null => {
+			if(typeVersion === undefined) {
+				typeVersion = getters.nodeTypeDefaultVersion(nodeType) || 1;
```

Comment:
magic number - might be clearer to move this to top of file `DEFAULT_TYPE_VERSION`